SlackPirate.py

```diff
@@ -186,6 +191,19 @@ def check_token_validity(token, user_agent: str) -> ScanningContext:
     return result
 
 
+def check_if_admin_token(token, output_info: ScanningContext):
+
+    try:
+        r = requests.get("https://slack.com/api/users.info", params=dict(token=token, pretty=1, user=output_info.user_id, headers={'User-Agent': output_info.user_agent})).json()
```

Comment:
Chop this long line - what IDE are you using? I recommend PyCharm community edition. It will highlight PEP8 problems such as overly long lines